cheatsheet.md

```diff
@@ -11,13 +11,11 @@ ChaiScript tries to follow the [Semantic Versioning](http://semver.org/) scheme.
 # Initializing ChaiScript
 
 ```
-chaiscript::ChaiScript chai; // loads stdlib from loadable module on file system
-chaiscript::ChaiScript chai(chaiscript::Std_Lib::library()); // compiles in stdlib
+chaiscript::ChaiScript chai; // initializes ChaiScript, adding the standard ChaiScript types (map, string, ...)
```

Comment:
Removing docs around Std_Lib here. Mind restoring them? It's helpful to know how to compile in stdlib.

Reply:
Probably I'm missing something here. I was looking at the `Chaiscript` constructor in `chaiscript.hpp`. The implementation is:

```c++
ChaiScript(std::vector<std::string> t_modulepaths = {},
    std::vector<std::string> t_usepaths = {},
    const std::vector<Options> &t_opts = chaiscript::default_options())
  : ChaiScript_Basic(
      chaiscript::Std_Lib::library(),
      std::make_unique<parser::ChaiScript_Parser<eval::Noop_Tracer, optimizer::Optimizer_Default>>(),
      t_modulepaths, t_usepaths, t_opts)
  {
  }
```

There is no option to pass in the `ModulePtr` generated by Std_Lib. Going from that, the example shouldn't work. I tried it, and in fact it doesn't compile. I guess this is a legacy implementation...?

Anyways, this doesn't matter, because Std_Lib is added by the default constructor.

Reply:
An example of its use is over at Spiced:
https://github.com/ChaiScript/Spiced/blob/master/src/chaiscript_creator.cpp

Reply:
Yeah, but this uses version 5.7.2 of Chaiscript. The current one does not support this behaviour. This documentation should therefore be removed.

Reply:
Ah, I see it now, thanks!